From 586bdd2e49e34127dc87de7f93b148ec3aeca0cc Mon Sep 17 00:00:00 2001
From: Joakim Bech <joakim.bech@linaro.org>
Date: Wed, 21 Feb 2018 10:09:21 +0100
Subject: [PATCH] Add template_ta files to root FS

Signed-off-by: Joakim Bech <joakim.bech@linaro.org>
---
 common.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/common.mk b/common.mk
index 7b1b4a5..80f3873 100644
--- a/common.mk
+++ b/common.mk
@@ -470,4 +470,6 @@ filelist-tee-common: optee-client xtest optee-examples
 	fi
 	@echo "file /etc/init.d/optee $(BUILD_PATH)/init.d.optee 755 0 0"	>> $(fl)
 	@echo "slink /etc/rc.d/S09_optee /etc/init.d/optee 755 0 0"	>> $(fl)
+	@echo "file /bin/template_ta $(ROOT)/template_ta/host/template_ta_host 755 0 0" >> $(fl)
+	@echo "file /lib/optee_armtz/11111111-2222-3333-445566778899aabb.ta $(ROOT)/template_ta/ta/out/11111111-2222-3333-445566778899aabb.ta 755 0 0" >> $(fl)
 	@echo "# filelist-tee-common /end"				>> $(fl)
-- 
2.11.0

